.github/workflows/deploy-production.yml

```diff
@@ -37,94 +36,118 @@ jobs:
     with:
       build_env: "production"
       deploy_env: "production"
-      cdk_stack: ${{ vars.WIDGET_STACK_NAME_PRODUCTION }}
+      S3_BUCKET: ${{ vars.S3_BUCKET_PRODUCTION }}
+      CF_DISTRIB_ID: ${{ vars.CF_DISTRIB_ID_PRODUCTION }}
     secrets:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      AWS_REGION: ${{ secrets.WIDGET_REGION_PRODUCTION }}
-      INFRA_CONFIG: ${{ secrets.INFRA_CONFIG_PRODUCTION }}
       SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
       SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
 
-  api-build:
+  api-prod:
     if: needs.files-changed.outputs.api == 'true'
     needs: files-changed
-    uses: ./.github/workflows/_reusable_build.yml
+    uses: ./.github/workflows/_deploy_api.yml
     with:
-      path: "api/app" # there's no "build" for lambdas, so one job for both works
-    secrets: inherit
-  lambdas-build:
-    if: needs.files-changed.outputs.lambdas == 'true'
-    needs: files-changed
-    uses: ./.github/workflows/_reusable_build.yml
+      deploy_env: "production"
+      ECR_REPO_URI: ${{ vars.ECR_REPO_URI_PRODUCTION }}
+      ECS_CLUSTER: ${{ vars.ECS_CLUSTER_PRODUCTION }}
+      ECS_SERVICE: ${{ vars.ECS_SERVICE_PRODUCTION }}
+      AWS_REGION: ${{ vars.API_REGION_PRODUCTION }}
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+  api-sandbox:
+    uses: ./.github/workflows/_deploy_api.yml
+    needs: [api-prod]
+    if: ${{ !failure() && needs.api-prod.result == 'success' }}
     with:
-      path: "api/lambdas"
-      install-from-parent: false
-    secrets: inherit
-  api-deploy:
-    uses: ./.github/workflows/_reusable_deploy.yml
-    needs: [api-build, lambdas-build]
-    if: ${{ !failure() && (needs.api-build.result == 'success' || needs.lambdas-build.result == 'success') }}
+      deploy_env: "sandbox"
+      ECR_REPO_URI: ${{ vars.ECR_REPO_URI_SANDBOX }}
+      ECS_CLUSTER: ${{ vars.ECS_CLUSTER_SANDBOX }}
+      ECS_SERVICE: ${{ vars.ECS_SERVICE_SANDBOX }}
+      AWS_REGION: ${{ vars.API_REGION_SANDBOX }}
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  infra-api-lambdas:
+    uses: ./.github/workflows/_deploy_cdk.yml
+    if: needs.files-changed.outputs.infra-lambdas == 'true'
```

Comment:
so used to seeing `===` in ts now that I thought this was a bug for split second 😆